src/main/java/com/parkingapp/parkingservice/domain/vehicle/Vehicle.java

```diff
@@ -0,0 +1,18 @@
+package com.parkingapp.parkingservice.domain.vehicle;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+public class Vehicle {
+    private UUID id;
+    private String brand;
+    private String model;
+    private String color;
```

Comment:
Maybe we could use an enum here, just the name e.g.: 

enum Color {
YELLOW, RED, PINK, BLACK
}